### PR TITLE
Main IPT migrated

### PR DESCRIPTION
**Pull request by rukayaj**

### CODE REVIEW

File: helm/ipt-s3/templates/deployment.yaml [hunk 1]
```diff
@@ -50,14 +50,14 @@ spec:
                 name: s3-secret
                 key: s3-secret-key
           - name: S3_ZIP_BUCKET_NAME
-            value: {{ .Values.zipBucket}}
+            value: {{ .Values.zipBucket }}
         resources:
           limits:
-            memory: 256Mi
-            cpu: "0.2"
+            memory: {{ .Values.sidecarLimitsMemory }}
```
Comment (rukayaj) [inline helm/ipt-s3/templates/deployment.yaml:56]: @MichalTorma do we still need this do you think?

Comment (MichalTorma) [reply helm/ipt-s3/templates/deployment.yaml:56]: I hope not... let's default to not having it and if we find some more issues (I know, highly unlikely). let's hope `mc` is not that hungry...

File: sidecar/setup.sh [hunk 2]
```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
+mc alias set sigma2 $S3_HOST $S3_ACCESS_KEY $S3_SECRET_KEY
+
 echo "Create the bucket if it doesn't exist"
 set +e
-s4cmd mb s3://$S3_BUCKET_NAME --endpoint-url $S3_HOST
+mc mb "sigma2/$S3_BUCKET_NAME"
```
Comment (MichalTorma) [inline sidecar/setup.sh:6]: so should we try `/root/minio-binaries/mc mb "sigma2/$S3_BUCKET_NAME"` here instead?

File: sidecar/sync.sh [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
 
 {
-    echo "bucket_name: $S3_BUCKET_NAME, S3_HOST: $S3_HOST, S3_SECRET_KEY: $S3_SECRET_KEY, S3_ACCESS_KEY: $S3_ACCESS_KEY."
-    echo "$(date '+%Y-%m-%d %T') Sync starting" >> /var/log/sync.log
-    # Sync from local to remote for backups
-    s4cmd dsync --recursive --sync-check --num-threads=3 /srv/ipt s3://$S3_BUCKET_NAME --endpoint-url $S3_HOST && \
-    echo "$(date '+%Y-%m-%d %T') Sync completed succesfully" >> /var/log/sync.log
-} >> /var/log/syncdebug.log 2>&1
+    echo "$(date '+%Y-%m-%d - %T') Sync starting"
+    mc mirror --overwrite --remove --preserve /srv/ipt sigma2/$S3_BUCKET_NAME
```
Comment (MichalTorma) [inline sidecar/sync.sh:5]: should we try `/root/minio-binaries/mc mirror --overwrite --remove --preserve /srv/ipt sigma2/$S3_BUCKET_NAME` ?

**Review by MichalTorma — APPROVED** [review]
Amazing!

**Pull request merged**